rpi5.yaml

```diff
@@ -340,23 +340,6 @@ parameters:
 
   DOMD_ROOT:
     desc: "Domd root device"
-    mmc:
-      overrides:
-        components:
-          domd:
-            builder:
-              conf:
-                - [DOMD_OVERLAYS, "%{SOC_FAMILY}-%{MACHINE}-mmc.dtbo"]
-                - [MACHINE_FEATURES:append, " domd_mmc"]
-        images:
-          full:
-            partitions:
-              rootfs:
-                gpt_type: B921B045-1DF0-41C3-AF44-4C6F280D3FAE # Linux aarch64 root
-                type: raw_image
-                size: 8 GiB
-                image_path: "%{YOCTOS_WORK_DIR}/%{DOMD_BUILD_DIR}/tmp/deploy/images/%{MACHINE}/%{XT_DOMD_IMAGE}-%{MACHINE}.rootfs.ext4"
-
     usb:
```

Comment:
Option with one variant makes no sense. `overrides` below should be moved to appropriate places in the main tree.

Reply:
We have left this option to leave a room for the future enhancement. for example Artem wanted nvme to be added. Also, IIRC option with only one value is common for moulin. (MACHINE for example)